tensorflow/compiler/mlir/lite/ir/tfl_ops.td

```diff
@@ -5411,6 +5411,9 @@ def TFL_Conv3DOp : TFL_Op<"conv_3d", [
 
 def TFL_Conv3DTransposeOp : TFL_Op<"conv_3d_transpose", [
     Pure,
+    QuantizableResult,
+    AffineQuantizedOpInterface,
+    AffineOpCoefficient<3, 1>,
```

Comment:
Apparently, there are more discussions for which the per channel dimension location should be supported in this operator in the long term.

Could you provide the rationale for your choice here?